lib/assert.js

```diff
@@ -278,16 +278,17 @@ function parseCode(code, offset) {
 }
 
 function getErrMessage(message, fn) {
-  const tmpLimit = Error.stackTraceLimit;
-  const errorStackTraceLimitIsWritable = isErrorStackTraceLimitWritable();
+  // eslint-disable-next-line node-core/prefer-primordials
+  const { stackTraceLimit } = Error;
```

Comment:
What about adding a `getStackTraceLimit` to `internal/errors` so we don't have to disable the ESLint rule everywhere?

Reply:
Thinking about it: I am not certain that's ideal as we have to just export `Error.stackTraceLimit` from `internal/errors` in that case by wrapping it in a method. We could just improve the rule itself to skip this property.

Reply:
Are you afraid of the perf cost of calling a method?
Having a "get" counterpart of the "set" that already exists seemed natural to me.

Reply:
I'm also fine with an exception to the rule.

Reply:
It's already an exception on the rule: https://github.com/nodejs/node/blob/fe514bf960ca1243b71657af662e7df29f5b57cf/lib/.eslintrc.yaml#L205-L208

If it's complaining, it means there's a bug with the rule that we should fix (hopefully before landing this PR). I'll take a look.

Reply:
Can you rebase now that the ESLint rule has been fixed?

Reply:
Done